src/Adapter/Customer/CommandHandler/AddCustomerHandler.php

```diff
@@ -130,8 +140,9 @@ private function fillCustomerWithCommandData(Customer $customer, AddCustomerComm
         $customer->optin = $command->isPartnerOffersSubscribed();
         $customer->birthday = $command->getBirthday()->getValue();
         $customer->id_shop = $command->getShopId();
+        $customer->is_guest = $command->isGuest();
```

Comment:
`$customer->is_guest` is `false` by default, `$command->isGuest()` will always be false now, except in tests.